pkg/webhook/pod/webhook.go

```diff
@@ -261,6 +281,33 @@ func MutateDaemonSet(ds *appsv1.DaemonSet, sliceName string) *appsv1.DaemonSet {
 	return ds
 }
 
+func (wh *WebhookServer) ValidateServiceExport(svcex *v1beta1.ServiceExport, ctx context.Context) (bool, string, error) {
+
+	log := logger.FromContext(ctx)
+	log.Info("fetching all serviceexport objects belonging to the slice", "slice", svcex.Spec.Slice)
+	serviceExportList, err := wh.SliceInfoClient.GetAllServiceExports(context.Background(), wh.Client, svcex.Spec.Slice)
+	if err != nil {
+		return false, "", err
+	}
+
+	newAliases := svcex.Spec.Aliases
+
+	for _, serviceExport := range serviceExportList.Items {
+		// In case we are updating an existing ServiceExport resource
+		if svcex.ObjectMeta.Name == serviceExport.ObjectMeta.Name &&
```

Comment:
Why are you skipping the existing service export updation validation?

Reply:
If we are updating a serviceexport object svcex-1:
1. serviceExportList will contain svcex-1.
2. Comparing the aliases of updated svcex-1 and existing svcex-1 may give us conflict.
3. Even if we are updating the aliases, it is safe to compare with aliases of other serviceexport objects not the same seviceexport object.